frontend/src/scenes/paths/pathsDataLogic.ts

```diff
@@ -147,6 +147,10 @@ export const pathsDataLogic = kea<pathsDataLogicType>([
                         label: path_dropoff_key || path_start_key || path_end_key || 'Pageview',
                         isDropOff: Boolean(path_dropoff_key),
                     }),
+                    additionalFields: ['event_count'],
+                    mapFields: (result: any[]) => ({
+                        value_at_data_point: result[2],
+                    }),
```

Comment:
Having given it some though, I'm not too happy with this. It feels like mixing concepts of different levels into one another. It's a bit weird for some frontend JS to know that 1) this query contains a hidden field "event_count", 2) it's the third element in the returned array, 3) the internal values used by the persons modal.

Any way to simplify it? Perhaps `value_at_data_point` should just always be a default returned field in an insight actors query, similar to `$session_id`? Perhaps the personsModalLogic should look for `record.value_at_data_point || record.event_count || record.value` or so?

Reply:
I also had the idea already to at least combine the field name and the mapping more. I'll think about it.

Reply:
Now it's just:

```
additionalFields: {
    value_at_data_point: 'event_count',
},
```

And the rest is done under the hood. And where you pass in the additional fields, you pass the actor query anyways, so it shouldn't be a great disconnect.

Reply:
Context where this comes in handy later: 

https://github.com/PostHog/posthog/pull/19959/files#diff-b6467c0a8333f0739d45f563b42a287900d7b96d7298236908d3283c38190c08